### PR TITLE
Support schema extraction in express response objects

### PR DESCRIPTION
**Pull request by simon-id**
### What does this PR do?

When a express response method (`send`, `json` or `jsonp`) is used with an object, sends the object to the waf to extract the schema and detect some sensitive values.

### Motivation

Supports response data in API Security

### CODE REVIEW

**Review by rochdev — DISMISSED** [review]

File: packages/dd-trace/test/appsec/index.express.plugin.spec.js [hunk 1]
```diff
@@ -96,20 +107,65 @@ withVersions('express', 'express', version => {
...
+      describe('with requestSampling 1.0', () => {
+        beforeEach(() => {
+          config.appsec.apiSecurity.requestSampling = 1.0
+          appsec.enable(config)
+        })
 
-        const expectedSchema = zlib.gzipSync(JSON.stringify([{ 'key': [8] }])).toString('base64')
-        const res = await axios.post(`http://localhost:${port}/`, { key: 'value' })
+        it('should get the schema', async () => {
+          const expectedRequestBodySchema = zlib.gzipSync(JSON.stringify([{ 'key': [8] }])).toString('base64')
```
Comment (simon-id) [inline packages/dd-trace/test/appsec/index.express.plugin.spec.js:117]:
```javascript
function formatSchema (obj) {
  return zlib.gzipSync(JSON.stringify(obj)).toString('base64')
}
```

File: packages/dd-trace/test/appsec/index.express.plugin.spec.js [hunk 2]
```diff
@@ -121,10 +177,11 @@ withVersions('express', 'express', version => {
         await agent.use((traces) => {
           const span = traces[0][0]
           expect(span.meta).not.to.haveOwnProperty('_dd.appsec.s.req.body')
+          expect(span.meta).not.to.haveOwnProperty('_dd.appsec.s.res.body')
         })
 
         expect(res.status).to.be.equal(200)
-        expect(res.data).to.be.equal('DONE')
+        expect(res.data).to.be.deep.equal('DONE')
```
Comment (simon-id) [inline packages/dd-trace/test/appsec/index.express.plugin.spec.js:184]: deep equal a string ? what

Comment (uurien) [reply packages/dd-trace/test/appsec/index.express.plugin.spec.js:184]: I didn't realize in the review... I made some test changing the `'DONE'` by and object... and forget to restore. Changing it..

File: packages/dd-trace/test/appsec/index.express.plugin.spec.js (hunk 1 above)
Comment (uurien) [reply packages/dd-trace/test/appsec/index.express.plugin.spec.js:117]: You're right, I don't have the habit to reuse code in tests... and I should

**Comment by simon-id** [comment]
ok LGTM

**Review by rochdev — APPROVED** [review]

**Review by CarlesDD — APPROVED** [review]

**Pull request merged**